src/host/registry.cpp

```diff
@@ -250,17 +250,14 @@ void Registry::LoadFromRegistry(_In_ PCWSTR const pwszConsoleTitle)
 
     // Now load complex properties
     // Some properties shouldn't be filled by the registry if a copy already exists from the process start information.
-    DWORD dwValue;
+    const auto loadDWORD = [=](const auto valueName) {

```

Comment:
Well I love it. But I do wonder if we should just be making it into a first-class function or template citizen instead of just a lambda. It sure cleans up the below calls.

Reply:
Yeah, I did consider this, but pulling it out into a first-class function means you need to pass additional parameters that you get for free with the lambda, so it's not quite as clean. I do think perhaps there's potential for a nicer registry encapsulation class here that could clean up more of the code, but I didn't want to get too sidetracked in this PR, and the lambda at least doesn't make things a whole lot worse.

Reply:
I almost broke down and worked on better-encapsulating the registry code some time a few years back when I first saw this file . . . so yeah, I get that for sure :D